scripts/shared/lib/deploy_helm

```diff
@@ -54,31 +54,37 @@ function helm_install_subm() {
         return
     fi
 
+    flags=(   --kube-context "${cluster}" \
+              --name submariner \
+              --namespace "${SUBM_NS}" \
+              --set "ipsec.psk=${SUBMARINER_PSK}" \
+              --set "broker.server=${submariner_broker_url}" \
+              --set "broker.token=${submariner_broker_token}" \
+              --set "broker.namespace=${SUBMARINER_BROKER_NS}" \
+              --set "broker.ca=${submariner_broker_ca}" \
+              --set "submariner.clusterId=${cluster}" \
+              --set "submariner.clusterCidr=${cluster_CIDRs[$cluster]}" \
+              --set "submariner.serviceCidr=${service_CIDRs[$cluster]}" \
+              --set "submariner.globalCidr=${global_CIDRs[$cluster]}" \
+              --set "serviceAccounts.globalnet.create=${globalnet}" \
+              --set "submariner.natEnabled=false" \
+              --set "routeAgent.image.repository=localhost:5000/submariner-route-agent" \
+              --set "routeAgent.image.tag=local" \
+              --set "routeAgent.image.pullPolicy=IfNotPresent" \
+              --set "engine.image.repository=localhost:5000/submariner" \
+              --set "engine.image.tag=local" \
+              --set "engine.image.pullPolicy=IfNotPresent" \
+              --set "globalnet.image.repository=localhost:5000/submariner-globalnet" \
+              --set "globalnet.image.tag=local" \
+              --set "globalnet.image.pullPolicy=IfNotPresent" \
+              --set "crd.create=${crd_create}" )
```

Comment:
I think for this small change it's a bit of an overkill to use an array.
You can (and IMO should) just set a local variable for the flag and pass that. You can see it in action here https://github.com/submariner-io/shipyard/blob/master/scripts/shared/lib/deploy_operator#L21